cluster/src/test/scala/com/evolutiongaming/cluster/AddressHelperSpec.scala

```diff
@@ -1,19 +1,20 @@
 package com.evolutiongaming.cluster
 
 import akka.actor.Address
-import org.scalatest.{FunSuite, Matchers}
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
 
-class AddressHelperSpec extends FunSuite with Matchers {
+class AddressHelperSpec extends AnyFunSpec with Matchers {
   val global = Address("akka.tcp", "coreservices", "127.0.0.1", 9196)
   val local = Address("akka", "coreservices")
   val addressHelper = new AddressHelper(global)
 
-  test("toLocal") {
+  it("toLocal") {
```

Comment:
what is `it`?

Reply:
the most basic way to define test case. `test` word disappeared from API

Reply:
I will check logs, if those test cases are run...

on similar project locally I do not see such funtion

Reply:
it was ran